# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/zstd.patch.ia64
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

ia64-t2-linux-gnu-ld: lib/zstd/common/error_private.o: linking constant-gp files with non-constant-gp files

--- linux-5.14/arch/ia64/Makefile.vanilla	2021-09-21 13:41:05.870356415 +0200
+++ linux-5.14/arch/ia64/Makefile	2021-09-21 13:42:15.501353639 +0200
@@ -25,6 +25,7 @@
 cflags-y	:= -pipe $(EXTRA) -ffixed-r13 -mfixed-range=f12-f15,f32-f127 \
 		   -falign-functions=32 -frename-registers -fno-optimize-sibling-calls
 KBUILD_CFLAGS_KERNEL := -mconstant-gp
+KBUILD_CFLAGS_MODULE += -mconstant-gp
 
 GAS_STATUS	= $(shell $(srctree)/arch/ia64/scripts/check-gas "$(CC)" "$(OBJDUMP)")
 KBUILD_CPPFLAGS += $(shell $(srctree)/arch/ia64/scripts/toolchain-flags "$(CC)" "$(OBJDUMP)" "$(READELF)")
